packages/client/src/__tests__/integration/happy/aggregations/test.ts

```diff
@@ -116,27 +116,27 @@ describe('aggregations', () => {
     } catch (err) {
       expect(err.message).toMatchInlineSnapshot(`
 
-                Invalid \`prisma.user.aggregate()\` invocation:
+        Invalid \`prisma.user.aggregate()\` invocation:
 
-                {
-                  where: {
-                    age: {
-                      gt: -1
-                    }
-                  },
-                  skip: 0,
-                  take: 10000,
-                  _avg: {
-                ?   age?: true,
-                    email: true
-                    ~~~~~
-                  }
-                }
+        {
+          _avg: {
+        ?   age?: true,
+            email: true
+            ~~~~~
+          },
+          where: {
+            age: {
+              gt: -1
+            }
+          },
+          skip: 0,
+          take: 10000
+        }
```

Comment:
Why the changed order here?

Reply:
I think it's because of the orders of the creation of the keys when we "de-sugar" user input.